### PR TITLE
[Ec2][Python][5R] Multiple if-else statement

### PR DESCRIPTION
**Pull request by eliottlv**
Issue #142

### CODE REVIEW

File: python-plugin/src/test/resources/checks/avoidUsingMultipleIfElseStatement.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        if nb1 == 1:
+            nb1 = 1
+        else:
+            print(nb1)
+        if nb1 == 1:# Noncompliant {{Use a match-case statement instead of multiple if-else if possible}}
+            nb1 = 1
+        else:
+            print(nb2)
+
+    def method_with_one_if_else_if():
```
Comment (dedece35) [inline python-plugin/src/test/resources/checks/avoidUsingMultipleIfElseStatement.py:29]: add new test with `if` and `else` separated instead of `elif`

**Review by dedece35 — CHANGES_REQUESTED** [review]
please add also updates with this rule implementation : 
- [`CHANGELOG.md`](https://github.com/green-code-initiative/ecoCode/blob/main/CHANGELOG.md) : add issue link
- [`RULES.md`](https://github.com/green-code-initiative/ecoCode/blob/main/RULES.md) : change icon

another modification to do : add a tests in [ecoCode-python-test-project](https://github.com/green-code-initiative/ecoCode-python-test-project) ==> please check others tests and make the same. as you can see test files are quite identical to unit test resources files

please make corrections on code smell problems (sonarcloud report)

**Comment by Affadine** [comment]
Match/case statement is not a performance tool although it is useful for pattern matching. For example if you need a first element test that needs to be type error proof and empty list proof, it seems that using a pattern match is faster than the semantically equivalent if block.
The multiple if else is faster for simple conditions but match/case is more beautiful, and for multiple checks, will be a clear winner.

**Comment by dedece35** [comment]
Hi @Affadine,
thank you for explanations. I agree with you, that's why the message of this rule present on `DESCRIPTION` variable explain us that isn't mandatory adn user can give its multiple if/else if wanted (for this, a `NOSONAR` could be appended to ignored error).
If you check the pythin test file, in that case, the match/case is preferred to multiple if/else.

@eliottlv, could you make review modifications to merge this PR, please ?
thank you.

**Comment by dedece35** [comment]
waiting for #200 (upgrade test cases)

**Comment by dedece35** [comment]
- #200 now ok with PHP
- new PR for python in new python repository `ecoCode-python` in PR https://github.com/green-code-initiative/ecoCode-python/pull/1
- new PR for specifications in PR https://github.com/green-code-initiative/ecoCode/pull/215
